MapboxNavigation/RouteMapViewController.swift

```diff
@@ -8,7 +8,7 @@ import Turf
 class ArrowFillPolyline: MGLPolylineFeature {}
 class ArrowStrokePolyline: ArrowFillPolyline {}
 
-class RouteMapViewController: UIViewController {
+class RouteMapViewController_v2: UIViewController {
```

Comment:
This rename looks like work towards #1502. Maybe we should keep it out of scope for this PR?